apps/mobile/src/queries/balance/use-total-balance.ts

```diff
@@ -0,0 +1,65 @@
+import { AccountId } from '@/models/domain.model';
+import {
+  useBitcoinAccountTotalBitcoinBalance,
+  useWalletTotalBitcoinBalance,
+} from '@/queries/balance/bitcoin-balance.query';
+import { useStxBalance } from '@/queries/balance/stacks-balance.query';
+import { useStacksSigners } from '@/store/keychains/stacks/stacks-keychains.read';
+
+import { Money } from '@leather.io/models';
+import { sumMoney } from '@leather.io/utils';
+
+interface TotalBalance {
+  btcBalance: Money;
+  btcBalanceUsd: Money;
+  stxBalance: Money;
+  stxBalanceUsd: Money;
+  totalBalance: Money;
+}
+
+export function useGetStacksAddresses(accountId?: AccountId) {
+  const signers = useStacksSigners();
+
+  if (!accountId) {
+    return signers.list.map(signer => signer.address);
+  }
+
+  const { fingerprint } = accountId;
+  return signers.list
+    .filter(signer => signer.descriptor.includes(fingerprint))
+    .map(signer => signer.address);
+}
+
+export function useWalletTotalBalance(): TotalBalance {
```

Comment:
I  think maybe this should change to plural `useWalletsTotalBalance` as it should support multiple wallets? 

Or else if we have a different name (like portfolio) to signify the users total portfolio balance - all wallets added to the app. 

@kyranjamie I asked a [Q here](https://github.com/leather-io/mono/pull/520/files#r1822656476) about `useWalletTotalBitcoinBalance` . 

I assume that includes all wallets the user has added as it's using their keychains